src/main/java/cashleh/commands/AddExpense.java

```diff
@@ -1,29 +1,22 @@
 package cashleh.commands;
 
-import cashleh.CashLehException;
 import cashleh.Expense;
+import cashleh.ExpenseStatement;
+import cashleh.IncomeStatement;
+import cashleh.Ui;
 
 public class AddExpense extends Command {
-    public static final String COMMAND = "addExpense";
     private final Expense expenseToAdd;
-
-    public AddExpense(String description, double amount) throws CashLehException {
-        if (description.isEmpty() || amount <= 0) {
-            throw new CashLehException("Invalid expense. Please include description and " +
-                    "amount of expense in following format:" +
-                    "\"addExpense <DESCRIPTION> /amt <AMOUNT>\"");
-        }
-        this.expenseToAdd = new Expense(description, amount);
-    }
+    private final Ui ui = new Ui();
```

Comment:
I think that we should change how the Ui class is used. It should not be initialised in the expense class. Perhaps we can change the methods it contain to static methods to prevent this unnecessary initialisation